templates/artist/includes/artist_detail_investors.html

```diff
@@ -5,13 +5,14 @@
             <li id="investor-block">
                 {% if investor.public_profile_url %}
                     <a href="{{ investor.public_profile_url }}">
-                        <img class="profile-crop" src="{{ investor.avatar_url }}" alt="{{ investor.name }}" />
-                        {{ investor.name }}
-                    </a>
+                        <img class="profile-crop-list" src="{{ investor.avatar_url }}" alt="{{ investor.name }}" />
+                        <br/>{{ investor.name }}:
+                    
                 {% else %}
                     {{ investor.name }}
                 {% endif %}
                 ${{ investor.total_investment }}
+                    </a>
```

Comment:
Moving the `</a>` tag here is wrong because if the investor doesn't have a public_profile_url, there will be an extra `</a>` tag generated here without an open tag.

Alternatively you can either copy line 14 into both `if` clauses, or surround the `</a>` with `{% if investor.public_profile_url %}` and `{% endif %}` tags.